### PR TITLE
[stable/metabase] release 0.31.2

### PR DESCRIPTION
**Pull request by ahmadalli**
Signed-off-by: ahmadali shafiee <mail@ahmadalli.net>

### CODE REVIEW

File: stable/metabase/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 image:
   repository: metabase/metabase
-  tag: v0.30.1
+  tag: v0.31.2
```
Comment (davidkarlsen) [inline stable/metabase/values.yaml:8]: update the readme too for consistency

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/metabase/values.yaml (hunk 1 above)
Comment (ahmadalli) [reply stable/metabase/values.yaml:8]: done

**Pull request merged**